### DISCUSSION TITLE
Icon Needed

### DISCUSSION THREAD

**Discussion by Strobeener** [chrisvel/tududi#47] [General]
Not a high priority as you are continuing to refine, but an Icon would be nice.  I'm putting this on my dashboard and an Icon wold be cool.

**Comment by chrisvel** [comment]
Of course. I guess you mean for the logo correct ?

**Reply by Foxhunter-de** [reply]
I think he is speaking of a favicon. I would like that too, because without, it does not look very nice in the bookmarks ;)

**Comment by cweijden** [comment]
A (fav)icon or logo is indeed helpfull in a dashboard and as favicon.  Though I am by no means creative in that field, I've let AI come up with some variants.  I can imagine that developer does not want to settle on a specific logo/icon design yet, but untill an official logo comes along I'm using the pomodori in a envelope icon below (as in: messaging a task) in my dashboard.
I'm trying to figure out where to put the favicon (/app/public isn't the spot I noticed), but have not yet succeeded with that.

![Tududi](https://github.com/user-attachments/assets/f6bb3267-8aea-4e9c-aaa0-5975ad972ffd)
![TUv](https://github.com/user-attachments/assets/fcee6537-31e8-43b6-8abf-e76c86ff0777)

**Comment by chrisvel** [comment]
thank you @cweijden! I actually added a very simple one until I have a more professional looking logo. I hope it helps. Added in `v0.67`.

**Comment by chrisvel** [comment]
favicon added in v0.86!
